sample_code/ep102/rev04/t.py

```diff
@@ -0,0 +1,39 @@
+class User(tuple):
+    _fields = ('name', 'age', 'location')
+
+    def __new__(cls, name, age, location):
+        return tuple.__new__(cls, (name, age, location))
+
+    @property
+    def name(self):
+        return self[0]
+
+    @property
+    def age(self):
+        return self[1]
+
+    @property
+    def location(self):
+        return self[2]
+
+    def __repr__(self):
+        return (
+            f'{type(self).__name__}('
+            f'name={self.name!r}, age={self.age!r}, location={self.location!r}'
+            ')'
+        )
+
+    def print_me(self):
+        print(f'{self.name} is {self.age} years old and from {self.location}')
```

Comment:
Included the `print_me` method as well, so the example can be functional, when running `python rev04/t.py`.
(I had to remove the other `User` definition as well in order to avoid name shadowing)